pkg/recovery/repair_test.go

```diff
@@ -169,11 +308,26 @@ func TestNewRepairHandler(t *testing.T) {
 			t.Fatalf("invalid address in receipt: expected %s received %s", c1.Address(), receipt.Address)
 		}
 
+		data = socChunk.Data()
```

Comment:
receipt checks are not needed since you conscruc t them

Reply:
it is needed as the receipt is constructed from the address that is send to PSS. If the Repairhandler sends a wrong chunk address.. then receipt will fail...